tests/installation/logs_from_installation_system.pm

```diff
@@ -44,4 +44,6 @@ sub run {
     $self->save_upload_y2logs();
 }
 
+sub post_fail_hook { }
```

Comment:
You could execute the SUPER::post_fail_hook unless get_var('ASSERT_Y2LOGS') but maybe the post_fail_hook really does not make sense in this module. but please put a comment above the overwrite why we think this is a good idea.

I also recommend to overwrite the flags to make this module actually *not* fatal so that the test continues which could be useful to see if the early die has any consequences.